sdk/face/ai-vision-face-rest/src/responses.ts

```diff
@@ -836,1001 +667,1211 @@ export interface GetLargeFaceListFacesDefaultResponse extends HttpResponse {
 }
 
-export interface CreatePersonDefaultResponse extends HttpResponse {
+export interface CreatePersonGroupDefaultResponse extends HttpResponse {
   status: string;
   body: FaceErrorResponseOutput;
-  headers: RawHttpHeaders & CreatePersonDefaultHeaders;
+  headers: RawHttpHeaders & CreatePersonGroupDefaultHeaders;
 }
 
-/** The final response for long-running createPerson operation */
-export interface CreatePersonLogicalResponse extends HttpResponse {
+/** The request has succeeded. */
+export interface DeletePersonGroup200Response extends HttpResponse {
   status: "200";
-  body: CreatePersonResultOutput;
 }
 
-export interface DeletePerson202Headers {
-  "operation-location": string;
+export interface DeletePersonGroupDefaultHeaders {
+  /** String error code indicating what went wrong. */
+  "x-ms-error-code"?: string;
 }
 
-/** A successful call returns an empty response body. The service has accepted the request and will start processing soon. The client can query the operation status and result using the URL specified in the 'Operation-Location' response header. The URL expires in 48 hours. */
-export interface DeletePerson202Response extends HttpResponse {
-  status: "202";
-  headers: RawHttpHeaders & DeletePerson202Headers;
+export interface DeletePersonGroupDefaultResponse extends HttpResponse {
+  status: string;
+  body: FaceErrorResponseOutput;
+  headers: RawHttpHeaders & DeletePersonGroupDefaultHeaders;
 }
 
-export interface DeletePersonDefaultHeaders {
+/** A successful call returns the Person Group's information. */
+export interface GetPersonGroup200Response extends HttpResponse {
+  status: "200";
+  body: PersonGroupOutput;
+}
+
+export interface GetPersonGroupDefaultHeaders {
   /** String error code indicating what went wrong. */
   "x-ms-error-code"?: string;
 }
 
-export interface DeletePersonDefaultResponse extends HttpResponse {
+export interface GetPersonGroupDefaultResponse extends HttpResponse {
   status: string;
   body: FaceErrorResponseOutput;
-  headers: RawHttpHeaders & DeletePersonDefaultHeaders;
+  headers: RawHttpHeaders & GetPersonGroupDefaultHeaders;
 }
 
-/** The final response for long-running deletePerson operation */
-export interface DeletePersonLogicalResponse extends HttpResponse {
+/** The request has succeeded. */
+export interface UpdatePersonGroup200Response extends HttpResponse {
   status: "200";
 }
 
-/** A successful call returns the person's information. */
-export interface GetPerson200Response extends HttpResponse {
+export interface UpdatePersonGroupDefaultHeaders {
+  /** String error code indicating what went wrong. */
+  "x-ms-error-code"?: string;
+}
+
+export interface UpdatePersonGroupDefaultResponse extends HttpResponse {
+  status: string;
+  body: FaceErrorResponseOutput;
+  headers: RawHttpHeaders & UpdatePersonGroupDefaultHeaders;
+}
+
+/** A successful call returns an array of Person Groups and their information (personGroupId, name and userData). */
```

Comment:
I wonder why this is not a paged operation if the response is an array?

Reply:
I think it's because the size of the returned array is limited by the API itself, which is up to 1000.
API ref: https://learn.microsoft.com/en-us/rest/api/face/person-directory-operations/get-dynamic-person-groups?view=rest-face-v1.2-preview.1&tabs=HTTP